### PR TITLE
Grammar, typos

### PR DESCRIPTION
**Pull request by endafarrell**
Small changes to improve links, grammar and fix typos.

### CODE REVIEW

File: docs/docs/Gear Up/CGM.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 * Medtronic (MiniMed Paradigm REAL-Time Revel or Enlite)
 
 
-The openaps toolkit currently primarily supports three different CGM systems: the Dexcom G4 Platinum system (with or without the [Share](http://www.dexcom.com/dexcom-g4-platinum-share) functionality), the newer Dexcom G5 Mobile system and the [Medtronic system](https://www.medtronicdiabetes.com/treatment-and-products/enlite-sensor). Other CGM or CGM-like devices (Libre) can also be used if the data is uploaded to Nightscout and the OpenAPS rig has Internet connectivity.
+The OpenAPS toolkit currently primarily supports three different CGM systems: the Dexcom G4 Platinum system (with or without the [Share](http://www.dexcom.com/dexcom-g4-platinum-share) functionality), the newer Dexcom G5 Mobile system and the [Medtronic system](https://www.medtronicdiabetes.com/treatment-and-products/enlite-sensor). Other CGM or CGM-like devices (Libre) can also be used if the data is uploaded to Nightscout and the OpenAPS rig has Internet connectivity.
```
Comment (scottleibrand) [inline docs/docs/Gear Up/CGM.md]: The openaps toolkit is lowercase. If you want to refer to OpenAPS more generally you could remove the toolkit here.

File: docs/docs/Gear Up/CGM.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ This refers to the Dexcom receiver hardware. Note that your Dexcom should be nea
 
 ### Pulling CGM data from the cloud
 
-Your OpenAPS implementation can also pull CGM data from a Nightscout site in addition to pulling from the CGM directly. You can find more documentation about pulling CGM data from a Nightscout site [here](https://openaps.readthedocs.io/en/latest/docs/While You Wait For Gear/nightscout-setup.html).
+Your OpenAPS implementation can also pull CGM data from a Nightscout site in addition to pulling from the CGM directly. You can find more documentation about pulling CGM data from a Nightscout site [here](https://openaps.readthedocs.io/en/latest/docs/While%20You%20Wait%20For%20Gear/nightscout-setup.html).
```
Comment (scottleibrand) [inline docs/docs/Gear Up/CGM.md]: Spaces in URLs generally have to be encoded as %20

Comment (endafarrell) [reply docs/docs/Gear Up/CGM.md]: Indeed. Without these "%20" the link was broken when read on readthedocs.io.

Comment (scottleibrand) [reply docs/docs/Gear Up/CGM.md]: Sorry, I got it backwards and thought you were removing them. Thanks.

**Pull request merged**